assets/sass/_base.sass

```diff
@@ -74,6 +74,11 @@ pre
 .thumbnail
     border: none
 
+.card-thumbnail
+  width: 100%	  width: 100%
+  height: 100%	  height: 100%
```

Comment:
Is this repetition intentional?

Reply:
nope, it was just a typo, probably when I merged my theme with your master branch. Should be fixed now

Reply:
do I have to send the PR once more? I have never edited a file after I sent a PR

Reply:
You should be able to push another commit to your branch, and this PR will account for it.